src/app/page.tsx

```diff
@@ -1,9 +1,10 @@
 "use client";
 
-import LandingPage from "@components/LandingPage";
+import LandingPage from "@components/landingPage";
```

Comment:
The file here is in Uppercase but here it's being imported in lowercase https://github.com/OlivierJM/sparked-next/blob/f40425d57ea790436cd91f5749a6c7ea77a1643d/src/components/LandingPage/index.tsx this fail the build

Reply:
<img width="1313" alt="Screenshot 2023-10-24 at 8 34 59 AM" src="https://github.com/OlivierJM/sparked-next/assets/11255454/e7f33d1d-6dd3-42c6-8910-3dbfcd979f6a">

Reply:
Please rename it to lowercase on the main branch . Notice that all folder names are in lowercase.

Reply:
@brianmuks it's not the main branch that has issue, it is this same branch.